weaviate/gql/get.py

```diff
@@ -83,7 +83,13 @@ def __init__(
         if not isinstance(class_name, str):
             raise TypeError(f"class name must be of type str but was {type(class_name)}")
         if properties is None:
-            properties = []
+            class_schema = client.Schema.get(class_name=class_name)
```

Comment:
you don'T have access to the client here.

What you could do:
- When instantiating query, you could pass the schema object as additional argument
- Then you should be able to do schema.get(class_name)